bbq/javatests/java/com/google/bbq/BroadcastQueryClientTest.java

```diff
@@ -58,6 +59,7 @@
     @org.junit.Before
     public void setUp() throws Exception {
         MockitoAnnotations.initMocks(this);
+        when(mockContext.getPackageName()).thenReturn("com.example.app");
```

Comment:
The Google proto3 implementation does not tolerate `null` being passed as an argument to a setter method (in this case, for setRequestingApp() on BroadcastQuery.Builder). The Square proto2 implementation was more tolerant, so the fact that there was no package name for the current app didn't cause an issue.